src/app/query/mempool/mempool.hooks.ts

```diff
@@ -36,21 +35,13 @@ function useAccountUnanchoredMempoolTransactions(address: string) {
 
 export function useCurrentAccountFilteredMempoolTransactionsState() {
   const address = useCurrentAccountStxAddressState();
-  if (!address)
-    logger.error(
-      `Attempting to fetch from mempool with no address in ${useCurrentAccountFilteredMempoolTransactionsState.name}`
```

Comment:
@kyranjamie part of the issue opened identified that the `useAccountMempool` query is running before the address is present. These errors are logging a lot in the console and perhaps confusing users, but I'm not sure we need them if we just disable the query until the address is present? Is this ok with you?

Reply:
![Screen Shot 2022-09-13 at 12 25 07 PM](https://user-images.githubusercontent.com/6493321/189968206-fb75fc3a-6564-44a9-a633-385380f4234e.png)